### PR TITLE
Update ryan-koch CAC client certificate record.

### PR DESCRIPTION
**Pull request by Ryan-Koch**
## Summary

I got a new CAC! This updates the old record.

[this article](tbd) explains more about the approach used.

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/main/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

### CODE REVIEW

File: migrations/app/secure/20230117191419_rkoch_cac.up.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+-- This migration allows a CAC cert to have read/write access to all orders and the prime API.
+-- The Orders API and the Prime API use client certificate authentication. Only certificates
+-- signed by a trusted CA (such as DISA) are allowed which includes CACs.
+-- Using a person's CAC as the certificate is a convenient way to permit a
+-- single trusted individual to interact with the Orders API and the Prime API. Eventually
+-- this CAC certificate should be removed.
+-- Updating sha256 for new cac
+UPDATE client_certs
+SET sha256_digest = '8e2a89d0e578ba80cde13b726be2ac0b12f3ca4519a27d2f793c3f1f6bfabc03',
+	subject = 'CN=ryan-koch,OU=DoD+OU=PKI+OU=CONTRACTOR,O=U.S. Government,C=US',
```
Comment (rogeruiz) [inline migrations/app/secure/20230117191419_rkoch_cac.up.sql:10]: Verifying that this subject is not the subject in S3. Thanks @Ryan-Koch

File: migrations/app/migrations_manifest.txt [hunk 2]
```diff
@@ -781,4 +781,5 @@
 20230105211952_app_demo_cert_migration.up.sql
 20230106201854_app_loadtest_cert_migration.up.sql
 20230106211757_app_exp_cert_migration.up.sql
+20230117191419_rkoch_cac.up.sql
 20230120165432_update_camp_lejeune_closeout_value.up.sql
```
Comment (rogeruiz) [inline migrations/app/migrations_manifest.txt:785]: Looks like these aren't in the right order, but it shouldn't affect things as the CAC SQL is only touching the `client_certs` table.

**Comment by rogeruiz** [comment]
Verifying contents of the SQL files across our environments and I'll report back in here.

**Comment by rogeruiz** [comment]
Hey @Ryan-Koch I see that you updated this across all environments except for **PRD**. **PRD** contains a stub file. Could you elaborate on why the exception was made for **PRD** and if all the other environments are indeed looking to get updated. Thanks!

File: migrations/app/migrations_manifest.txt (hunk 2 above)
Comment (rogeruiz) [reply migrations/app/migrations_manifest.txt:785]: Nevermind on this, I see that you did a merge after creating the branch. Only calling it out. This isn't an issue.

**Review by rogeruiz — APPROVED** [review]
lgtm 🌈

I don't see any issues with what you're adding. I left some questions in the main thread to be answered. But this looks fine to add.

**Comment by Ryan-Koch** [comment]
> Hey @Ryan-Koch I see that you updated this across all environments except for **PRD**. **PRD** contains a stub file. Could you elaborate on why the exception was made for **PRD** and if all the other environments are indeed looking to get updated. Thanks!

This is a good question. The way we've handled these to date is to avoid putting personal CACs on prod, I presume on the idea that we don't want us to be able to act as a prime in that environment. So the stub file keeps things from breaking but enables us to make that distinction.

**Pull request merged**